### PR TITLE
fix: make the argument of configureKeyboardsPreference BOOL 

### PR DESCRIPTION
**Pull request by KazuCocoa**
According to https://github.com/appium/appium/issues/13360, Xcode 11 raises an error in some environments.

### CODE REVIEW

File: WebDriverAgentLib/Utilities/FBConfiguration.m [hunk 1]
```diff
@@ -312,16 +315,16 @@ + (void)configureKeyboardsPreference:(nonnull NSValue *)value forPreferenceKey:(
...
+      [controller setValue:@(enable) forPreferenceKey:FBKeyboardAutocorrectionKey];
     }
   } else if ([key isEqualToString:FBKeyboardPredictionKey]) {
     // Predictive in Keyboards
     if ([controller respondsToSelector:@selector(setPredictionEnabled:)]) {
-      controller.predictionEnabled = value;
+      controller.predictionEnabled = enable;
     } else {
-      [controller setValue:value forPreferenceKey:FBKeyboardPredictionKey];
+      [controller setValue:@(enableWebDriverAgentLib/Utilities/FBConfiguration.m) forPreferenceKey:FBKeyboardPredictionKey];
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Utilities/FBConfiguration.m]: Typo here?

File: WebDriverAgentLib/Utilities/FBConfiguration.m [hunk 2]
```diff
@@ -213,9 +213,12 @@ + (void)configureDefaultKeyboardPreferences
 
   TIPreferencesController *controller = [controllerClass sharedPreferencesController];
   // Auto-Correction in Keyboards
+  // 'setAutocorrectionEnabled' has been in TextInput.framework/TIKeyboardState.h over iOS 10.3
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Utilities/FBConfiguration.m]: perhaps `has been` -> `was`

File: WebDriverAgentLib/Utilities/FBConfiguration.m (hunk 1 above)
Comment (KazuCocoa) [reply WebDriverAgentLib/Utilities/FBConfiguration.m]: yeah... oops...

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by KazuCocoa** [comment]
This change worked (I checked).
Let me merge this and ask this if this fixes the error.

**Pull request merged**